corehq/apps/app_manager/static/app_manager/json/template_apps/case_management.json

```diff
@@ -634,53 +509,150 @@
               },
               "advanced": ""
             }
+          ],
+          "lookup_action": null,
+          "display": "long",
+          "lookup_enabled": false,
+          "lookup_responses": [
+            
+          ]
+        }
+      },
+      "doc_type": "Module",
+      "name": {
+        "en": "Registration"
+      },
+      "case_type": "case",
+      "task_list": {
+        "doc_type": "CaseList",
+        "label": {
+          
+        },
+        "media_audio": null,
+        "media_image": null,
+        "show": false
+      },
+      "fixture_select": {
+        "xpath": "",
+        "doc_type": "FixtureSelect",
+        "variable_column": null,
+        "fixture_type": null,
+        "active": false,
+        "localize": false,
+        "display_column": null
+      },
+      "media_audio": null,
+      "unique_id": "37fa1ba361352016c277661662eaca964c2e9cf2"
```

Comment:
They're okay as is: template apps use `import_app`, which goes through `export_json` > `scrub_source` > `update_unique_ids`